crates/api_types/src/service/kafka_connect.rs

```diff
@@ -0,0 +1,14 @@
+use serde::{Deserialize, Serialize};
+use std::collections::BTreeMap;
+use utoipa::ToSchema;
+
+/// Configuration for accessing a Kafka Connect service.
+#[derive(Debug, Clone, Eq, PartialEq, Serialize, Deserialize, ToSchema)]
+pub struct KafkaConnectServiceConfig {
```

Comment:
Doesn't Kafka Connect also have an HTTP endpoint used to manage its connectors?

Reply:
I've for now removed this connector type as well to limit the scope of the PR.

Reply:
I've for now removed this connector type as well to limit the scope of the PR.